### DISCUSSION TITLE
Request: Support for Docker container install

### DISCUSSION THREAD

**Discussion by nigiriemoji** [powenn/AltServer-Linux-PyScript#8] [Ideas]
Hello,

Is it possible to support docker containers for AltServer-Linux-PyScript? I would like to add it to my docker-compose.yml file

Thank you

**Comment by powenn** [comment]
For reference 

https://github.com/Macleykun/Altcon

**Comment by joshua-pe** [comment]
This would be a great addition to simplify starting this program. Although the main python program works great, I haven't been able to figure out how to automate its startup when the system boots. I have tried using a systemd service without success. (Although insecure, I hard-coded the sudo password to skip the prompt) 
Is there any documentation available that explains how to start this script on boot?

**Reply by powenn** [reply]
only netmuxd requires run as root, so if you can make netmuxd as an service that running as root on boot,then run a modified python script that won't asking  sudo password as another service

**Comment by powenn** [comment]
Possible will add support for docker in future :)
